src/applications/disability-benefits/526EZ/components/createDisabilityIncreaseApplicationEntry.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+
+const disabilityForms = new Set(['21-526EZ']);
+
+export default function createDisabilityIncreaseApplicationEntry(store) {
+  const root = document.getElementById('react-applicationEntry');
+  if (root) {
+    import(
+      /* webpackChunkName: "education-application-status" */
```

Comment:
You should change this chunk name, so it stays separate from the edu application bundle.

Reply:
Thanks @jbalboni, would "disability-application-status" work?

Reply:
Sounds good to me!